contrib/test/integration/vars.yml

```diff
@@ -31,3 +32,33 @@ ssh_location: "{{ lookup('env','HOME') }}/.ssh/id_rsa"
 # Additional environment variables for integration tests w/ userNS enabled
 userns_int_test_env:
     TEST_USERNS: "1"  # Consumed by test/test_runner.sh, by way of Makefile
+
+kata_int_test_env:
+    CONTAINER_RUNTIMES: "containerd-shim-kata-v2:/usr/bin/containerd-shim-kata-v2:/run/vc:vm"
+    CONTAINER_RUNTIME: "containerd-shim-kata-v2"
+    CONTAINER_DEFAULT_RUNTIME: "containerd-shim-kata-v2"
+    RUNTIME_NAME: "containerd-shim-kata-v2"
+    RUNTIME_ROOT: "/run/vc"
+    RUNTIME_TYPE: "vm"
+    TESTFLAGS: "ctr.bats"
+    JOBS: "1"
```

Comment:
why only one job here? can kata runtime not handle parallelization of more jobs?

Reply:
Well, you are right, it should... parallelization was disabled when the tests were run using devicemapper as the storage driver, but here it is not the case any more.
So, just tried to run the test enabling back parallelization... jobs started to fail :-( 
We need to disable parallelization for now, but to understand why some more investigation is needed.

Reply:
can you throw a TODO in there to not loose track of it, and upon this merging open an issue referencing the TODO?

Reply:
Sure, nice advice! Done